snap/snapcraft.yaml

```diff
@@ -0,0 +1,32 @@
+name: asimov
+base: core22
+version: '25.0.0-dev.3'
+summary: ASIMOV CLI Snap
+description: "Bundles a prebuilt ASIMOV CLI binary (gzipped) into a snap."
+
+grade: devel
+confinement: classic
+
+parts:
+  asimov-cli:
+    plugin: dump
+    source: .
+
+    override-pull: |
+      snapcraftctl pull
+
+      wget -O asimov-linux-x86.gz \
```

Comment:
This should probably be using `$ARCH` or `uname -m`, to support Linux on ARM as well.

Reply:
@artob it is done